src/utils/sortTypes.ts

```diff
@@ -0,0 +1,16 @@
+export const sortTypes = {
```

Comment:
Could use a better name, and maybe a test if we can't type it yet.

Reply:
That is named after a parameter in `react-table`. I'll add a jsdoc description